src/Router.js

```diff
@@ -1,18 +1,21 @@
 import React from 'react';
 import {Scene, Router, Actions, ActionConst} from 'react-native-router-flux';
-import {LoginForm} from "./LoginForm";
-import {TodayScreen} from "./TodayScreen";
-import {TomorrowScreen} from "./TomorrowScreen";
-import {ProblemScreen} from "./ProblemScreen";
+import firebase from 'firebase';
+import LoginForm from './LoginForm';
+import SignUpForm from './SignUpForm';
+import {TodayScreen} from './TodayScreen';
+import {TomorrowScreen} from './TomorrowScreen';
+import {ProblemScreen} from './ProblemScreen';
 
 export const RouterComponent = () => {
     return(
         // sceneStyle will be applied to all different scenes of our App
         <Router sceneStyle={{ paddingTop: 65 }}>
 
-                <Scene key="login" component={LoginForm} title="Please Login"/>
+                <Scene key="login" component={LoginForm} title="Log in to your account"/>
+                <Scene key="signup" component={SignUpForm} title="Sign up for free"/>
 
-                <Scene key="today" component={TodayScreen} title="Step 1/3" onRight={() => Actions.tomorrow()} rightTitle="Next" />
+                <Scene key="today" component={TodayScreen} title="Step 1/3" onBack={() => {firebase.auth().signOut(); Actions.login(); console.log('Signed out')}} onRight={() => Actions.tomorrow()} rightTitle="Next" />
```

Comment:
this should be move to separate method. Moreover move it to separate log out button, pls

Reply:
Deleted due to rearranging app's screens.